src/logic/calculate.js

```diff
@@ -0,0 +1,24 @@
+import operate from './operate';
+
+function calculate(calculatorDataObj, buttonName) {
+  let { total, next, operation } = calculatorDataObj;
+  if (buttonName === 'AC') {
+    total = '';
+    next = '';
+    operation = '';
+  } else if (buttonName === '+/-') {
+    total = next * total * -1;
```

Comment:
if button's name == "+/-", total and next should be multiplied by -1 to make it negative or positive. 

for example:
```

 if (buttonName === '+/-') {    
         return {      
           ...obj,     
              total: total * -1,     
             next: next * -1,    
           }; 
      }
```